docs/en/engines/table-engines/integrations/arrowflight.md

```diff
@@ -14,17 +14,20 @@ This integration allows ClickHouse to fetch data from external Flight-enabled se
 ## Creating a Table {#creating-a-table}
 
 ```sql
-CREATE TABLE [IF NOT EXISTS] [db.]table_name
-(
-    name1 [type1],
-    name2 [type2], ...
-) ENGINE = ArrowFlight('host:port', 'dataset_name');
+CREATE TABLE [IF NOT EXISTS] [db.]table_name (name1 [type1], name2 [type2], ...)
+    ENGINE = ArrowFlight('host:port', 'dataset_name' [, NOAUTH]);
+
+CREATE TABLE [IF NOT EXISTS] [db.]table_name (name1 [type1], name2 [type2], ...)
+    ENGINE = ArrowFlight('host:port', 'dataset_name', 'username', 'password');
 ```
 
 **Engine Parameters**
 
 * `host:port` — Address of the remote Arrow Flight server.
 * `dataset_name` — Identifier of the dataset on the Flight server.
+* `username` - Username to use with basic HTTP style authentication.
+* `password` - Password to use with basic HTTP style authentication.
+* NOAUTH - Specifies that authentication is not used. That will work only if the Arrow Flight server allows it.
```

Comment:
Do we need the `NOAUTH` param? Isn't not specifying the username and password enough? I think I never seen such argument in ClickHouse, though I might be wrong.

Reply:
done